pythonPackages.pywatchman: 1.3.0 -> 1.4.1

marsam opened the pull request:
###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

marsam committed:
pythonPackages.pywatchman: 1.3.0 -> 1.4.1

Mic92 commented on pkgs/top-level/python-packages.nix:
```diff
@@ -4431,17 +4431,15 @@ in {
 
   pywatchman = buildPythonPackage rec {
```
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.

marsam replied:
updated :)

marsam committed:
pythonPackages.pywatchman: move to python-modules

pywatchman is licensed under BSD-3

    https://github.com/facebook/watchman/commit/150f1fb4eff94c0248b54154fb356f768a131d60

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45732&attempt_id=5a124d7a-1ef2-4558-9cf7-ca5064daffd4)

Attempted: python2.pkgs.pywatchman, python3.pkgs.pywatchman

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking RPATHs of ELF executables and libraries in /nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1
shrinking /nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1/lib/python3.6/site-packages/pywatchman/bser.cpython-36m-aarch64-linux-gnu.so
strip is /nix/store/y4ymnvgxygpq05h03kyzbj572zmh6zla-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1/lib  /nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1/bin
patching script interpreter paths in /nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1
checking for references to /build in /nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1...
wrapping `/nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1/bin/watchman-make'...
wrapping `/nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1/bin/watchman-wait'...
/nix/store/99gsqhk8v7437bky1ls3fdshzmz87cms-python2.7-pywatchman-1.4.1
/nix/store/kvvcc39frhg3kb2ryhsyiliqpknnn9li-python3.6-pywatchman-1.4.1
```
</p></details>

GrahamcOfBorg commented:
No attempt on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45732&attempt_id=216699f5-b8d2-4b8f-956a-c8ba34b4dfbd)

The following builds were skipped because they don't evaluate on x86_64-linux: python2.pkgs.pywatchman, python3.pkgs.pywatchman

<details><summary>Partial log (click to expand)</summary><p>

```
Cannot nix-instantiate `python2.pkgs.pywatchman' because:
GC Warning: Failed to expand heap by 10737418240 bytes
Can't start up: not enough memory

Cannot nix-instantiate `python3.pkgs.pywatchman' because:
GC Warning: Failed to expand heap by 10737418240 bytes
Can't start up: not enough memory

```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45732&attempt_id=2da1399a-7d55-4436-a154-accd8c9d1704)

Attempted: python2.pkgs.pywatchman, python3.pkgs.pywatchman

<details><summary>Partial log (click to expand)</summary><p>

```
wrapping `/nix/store/fsw6831rpgydsym0pbpli693c9y88r28-python2.7-pywatchman-1.4.1/bin/watchman-make'...
/private/tmp/nix-build-python3.6-pywatchman-1.4.1.drv-0/pywatchman-1.4.1
post-installation fixup
strip is /nix/store/df6k4mgdjxciy0f637lryp7c9ln7n1m3-cctools-binutils-darwin/bin/strip
stripping (with command strip and flags -S) in /nix/store/bm5j1fb0h7p17wxlsg4aa44269c72crr-python3.6-pywatchman-1.4.1/lib  /nix/store/bm5j1fb0h7p17wxlsg4aa44269c72crr-python3.6-pywatchman-1.4.1/bin
patching script interpreter paths in /nix/store/bm5j1fb0h7p17wxlsg4aa44269c72crr-python3.6-pywatchman-1.4.1
wrapping `/nix/store/bm5j1fb0h7p17wxlsg4aa44269c72crr-python3.6-pywatchman-1.4.1/bin/watchman-wait'...
wrapping `/nix/store/bm5j1fb0h7p17wxlsg4aa44269c72crr-python3.6-pywatchman-1.4.1/bin/watchman-make'...
/nix/store/fsw6831rpgydsym0pbpli693c9y88r28-python2.7-pywatchman-1.4.1
/nix/store/bm5j1fb0h7p17wxlsg4aa44269c72crr-python3.6-pywatchman-1.4.1
```
</p></details>
